virt-attaching-vm-to-sriov: fix yaml example (#26322)
openshift/openshift-docs: modules/virt-attaching-vm-to-sriov-network.adoc

  domain:
    devices:
      interfaces:
      - name: default <1>
        masquerade: {} <2>
      - name: nic1 <3>
        sriov: {}
  networks:
  - name: default <4>
    pod: {}
  - name: nic1 <5>
    multus:
        networkName: <sriov-network> <4>
...
----

> <1> A name for the interface that is connect to the Pod network.

Reviewer: nit: 
```suggestion
<1> A name for the interface that is connected to the Pod network.
```

<2> The `masquerade` binding to the default Pod network.

> <3> A name for the SR-IOV interface.

Reviewer: To keep the articles consistent:
```suggestion
<3> The name for the SR-IOV interface.
```

Author: my use of "a" is intentional. "the name" alludes to a name that already exists somewhere else. But here, it is just a random name defined for this VM only and exists nowhere else.

Reviewer 2: In that case we tend to use "A unique name for .." to clarify that the user is defining the name. If that works for you then please apply to L34 and L36
We should also wrap the names in <names> in the config but I'll raise a separate comment above.

> <4> the name of the interface used in <1>.

Reviewer: ```suggestion
<4> The name of the interface used in <1>.
```

> <5> the name of the SR-IOV interface. This must be the same as <3>.

Reviewer: ```suggestion
<5> The name of the SR-IOV interface. This must be the same as <3>.
```

Author: thanks, fixed.

> <4> The name of the interface used in <1>.
> <5> The name of the SR-IOV interface. This must be the same as <3>.

Reviewer 2: Unfortunately the second callouts here don't build. 
```suggestion
<4> The name of the Pod network interface. This must be the same as the `interfaces.name` that you defined earlier.
<5> The name of the SR-IOV interface. This must be the same as the `interfaces.name` that you defined earlier..
```

Reviewer 2: @dankenigsberg - Did you want to take this suggestion or rewrite to avoid use of the second `<1>`/`<3>` in the callouts ?
That's the only thing preventing merge atm

Author: I missed your former comment. Can we not refer to the precise number? We have two `intrefaces.name` and we need to differenciate between them.

Author: but I cannot find a proper text right now, Ill take yours!

> <4> The name of the SR-IOV network attachement definition.

Reviewer 2: typo: s/attachement/attachment

Author: taken

> <4> The name of the SR-IOV network attachment definition.

Reviewer 3: @dankenigsberg, \<4> appears twice; Should it?

Author: certainly not. fixed.
